docs/Installation.md

```diff
@@ -42,6 +38,25 @@ mkdir homebrew && curl -L https://github.com/Homebrew/brew/tarball/master | tar
 ### Multiple installations
 Create a Homebrew installation wherever you extract the tarball. Whichever `brew` command is called is where the packages will be installed. You can use this as you see fit, e.g. a system set of libs in `/usr/local` and tweaked formulae for development in `~/homebrew`.
 
+### OS X Mountain Lion (10.8) and below
+Because GitHub now only allows clients that support TLS 1.2 to access repositories, systems running anything older than OS X Mavericks (10.9) will need their Homebrew installation to be completed manually.
+
+1. Using a newer web browser or a separate machine, download the files at the following URLs (do not use Safari if its *Open "safe" files after downloading* option is enabled) and move each to `~/Downloads`:
```

Comment:
Great work compiling these instructions. They kinda make me weep though so I'm wondering about modifying the installer to make life a bit easier. Could these be downloaded over an HTTP link, I wonder, and that be an optional parameter that's used? Alternatively on some versions we could use the system Git and the `git://` protocol perhaps?

Reply:
Each of the below GitHub download links always performs a 301 redirect to HTTPS if the request is made with HTTP, preventing < 10.9 from downloading with any built-in tools. But it turns out that checking out via SSH still works, which can be used on any system whose Xcode/CLT ships with Git (i.e. Snow Leopard / Xcode 4 and up), as long as its public key auth is already set up. It also turns out that Bintray still allows downloads over HTTP. I've revised these steps and they seem to test fine so far, although full testing will be delayed since my ESXi Mac's SSD just went bad. 🙈

Alternatively, if there was an official HTTP mirror for snapshots of `brew` and `homebrew-core`, the installer could be modified to grab those instead on < 10.9.

Reply:
`git://` is preferable to `ssh` because of the lack of setup that's required. I reckon I (or you) could adjust the installer accordingly to automate some of what's mentioned here.

Reply:
Take a look at the current state of Homebrew/install#154, in which installation is automated on < 10.9 as long as Git is preinstalled. If that's the approach you're after, this PR can be revised to eliminate the manual instructions.

Reply:
@EricFromCanada Yeh, I definitely reckon something like that is a step in the right direction. I don't think it's unreasonable to expect a bit more manual intervention to be required to make the official installer work (i.e. preinstalling Xcode/CLT and/or Git) if it let's us simplify assumptions elsewhere 👍